integration/stability/soak_parallel_rm.sh

```diff
@@ -106,9 +106,10 @@ check_all_running() {
 		fi
 
 		# check we have the right number of qemu's
-		how_many_qemus=$(pgrep -a -f ${HYPERVISOR_PATH} | wc -l)
-		if (( ${how_many_running} != ${how_many_qemus} )); then
-			echo "Wrong number of qemus running (${how_many_running} != ${how_many_qemus}) - stopping"
+		vmm_process=$(basename ${HYPERVISOR_PATH})
```

Comment:
note: you can use the bashism `${x##*/}` or similar instead of invoking basename :-)